jest.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
     preset: 'ts-jest',
     testEnvironment: 'node',
     testMatch: ['**/*.test.ts'],
+    testTimeout: 150000,
```

Comment:
it is neccesary for the tests to work ?

Reply:
Yes, the default timeout is 5 seconds.
Since those tests are integration tests and need an active connetion to the DB, they require more time.
You can see that i moved this configuration from the `startTestServer` method in `TestServer`, because this shouldn't be set programmatically.